app/src/main/java/com/example/githubusers/screens/main/StartAdapter.kt

```diff
@@ -28,6 +28,10 @@ class StartAdapter(var callBack: (Bundle) -> Unit) : ListAdapter<UsersItemEntity
 
     override fun onBindViewHolder(holder: ViewHold, position: Int) {
 
+        if (position >= itemCount-15) {
+            UpdateList()
+        }
```

Comment:
Very tricky. Better is to find last visible element with a help of scroll listener
Such logic should never be in adapter, it should be as simple as possible
Adapter should never give commands to ViewModel